recipes/hamilflow/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "hamilflow" %}
+{% set version = "0.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hamilflow-{{ version }}.tar.gz
+  sha256: 06e67d49480d876491c8fe52f5919a152c516ae6df50729ab05eb9e3ebcc3f9a
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<3.13
+    - poetry
+  run:
+    - loguru >=0.7.2,<0.8.0
+    - matplotlib-base >=3.8.3,<4.0.0
+    - numpy >=1.26.3,<2.0.0
+    - pandas >=2.0.0,<=2.2.0
+    - pydantic >=2.6.1,<3.0.0
+    - python >=3.6,<3.13
+
+test:
+  imports:
+    - hamilflow
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/hamilflow/
```

Comment:
Is there a GitHub repo as well or is this the only link?

Reply:
We have a GitHub repo
https://github.com/kausalflow/hamilflow

I will add the github repo instead.